### PR TITLE
feat: use TOML config

### LINKED ISSUE #20: Outdated Format
I think Alacritty uses .toml now?

### PR DESCRIPTION
**Pull request by nekowinston**
Closes #20.

This should be merged whenever Alacritty releases a version that includes https://github.com/alacritty/alacritty/commit/bd4906722a1a026b01f06c94c33b13ff63a7e044.

We don't need to keep the YAML files in the repo, as we can simply link to an older revision of this repo if people want to have a look at the YAML formats.

### CODE REVIEW

**Comment by nekowinston** [comment]
I'll add a

> # Usage
> **Note**
> If you're using Alacritty <`next Alacritty version`, please look at the [YAML tag](#yaml-tag)

after merging this.

File: README.md [hunk 1]
```diff
@@ -18,19 +18,27 @@
...
 2. Import the desired flavour config in your `alacritty.yml`:
 ```yaml
-import:
+import = [
   # uncomment the flavour you want below:
-  - ~/.config/alacritty/catppuccin/catppuccin-mocha.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-macchiato.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-frappe.yml
-#  - ~/.config/alacritty/catppuccin/catppuccin-latte.yml
+  "~/.config/alacritty/catppuccin/catppuccin-mocha.yml"
```
Comment (gersonsosa) [inline README.md:37]: this should end with .toml

**Review by gersonsosa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nekowinston) [reply README.md]: Good catch, thank you.

**Comment by nekowinston** [comment]
Seems that v0.13 is getting ready for release: https://github.com/alacritty/alacritty/releases/tag/v0.13.0-rc1

**Pull request merged**